frontend/src/views/SQRAllotment.vue

```diff
@@ -158,11 +157,7 @@ export default {
 
         this.$bindAsArray(
           "sqrFiles",
-          fb
-            .database()
-            .ref(`sqr/files/${this.filter.list}`)
-            .orderByChild("status")
-            .equalTo("Spare"),
+          fb.database().ref(`files/${this.filter.list}`),
           null, // cancel callback not used
           this.filterSelectedFiles
         );
```

Comment:
The new data structure does not allow that query since that status is now stored in the soundQualityReporting field.

Reply:
Status is there, see https://trello.com/c/VzBdmI43/13-data-structure.
Or maybe I'm misunderstanding something.

![image](https://user-images.githubusercontent.com/6065071/48406513-b71cf000-e75a-11e8-8685-12a9b236427a.png)

Reply:
Oh sorry, I read “not stored”. Can you elaborate on the restriction filtering by a sub-field?

Reply:
https://firebase.google.com/docs/database/rest/retrieve-data#filtering-by-a-specified-child-key

> Firebase also supports queries ordered by deeply nested children, rather than only children one level down.

Reply:
That works on entries that have a soundQualityReporting field already but you mentioned some may be left blank and it doesn't work on those.